CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Total cells column no longer required for vi-cell XR
 - Ignore invalid first row when present for vi-cell XR files
+- Capture concentration on Nanodrop Eight files that do not have NA Type column
 - Removed hardcoding of date parsing around Gen5 plate numbers
+
```

Comment:
Could you please remove this newline? Although I agree this makes things easier to read in plaintext, when viewed in markdown the newline isn't necessary. Not to mention internal style consistency is important.